exclude-from.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+set -e
+
+# Determine the paths to exclude from the deployment
+#
+# Paths are inferred from the REMOTE_PATH environment variable.
+#
+# The function returns two paths: the relative base path for prefixing excluded files,
+# and the relative mu-plugins path.
+determine_exclude_paths() {
+  local base_path="wp-content/"
+  local mu_dir_path="wp-content/mu-plugins/"
+
+  case "$REMOTE_PATH" in
+    "wp-content")
+      base_path=""
+      mu_dir_path="mu-plugins/"
+      ;;
+    "wp-content/mu-plugins")
+      base_path=""
+      mu_dir_path=""
+      ;;
+  esac
+
+  printf "%s\n%s\n" "$base_path" "$mu_dir_path"
+}
+
+# Generate a list of files to exclude from the deployment
+#
+# The list is output to stdout, which can be used as an argument to --exclude-from.
+generate_exclude_from() {
+  {
+    IFS= read -r base_path
+    IFS= read -r mu_dir_path
+  } <<< "$(determine_exclude_paths)"
```

Comment:
This is a shorthand approach for assigning local variables that respect `""` as a valid value.

In other words, calling `read -r base_path mu_dir_path "$(determine_exclude_paths)"` can overwrite the first value with the second if only the first value will be evaluated as `""`.